ios/RNSentry.m

```diff
@@ -171,33 +171,27 @@ - (void)setEventEnvironmentTag:(SentryEvent *)event
                   rejecter:(RCTPromiseRejectBlock)reject)
 {
     NSLog(@"Bridge call to: deviceContexts");
-    NSMutableDictionary<NSString *, id> *contexts = [NSMutableDictionary new];
+    __block NSMutableDictionary *contexts;
```

Comment:
What's the reason to make it a not inited `__block`?

Reply:
Yes, it's for the assigned dictionary in the configureScope to be visible outside that block so we can pass it to resolve.